### PR TITLE
Using new graph images for edit gas fee popover

### PR DESCRIPTION
**Pull request by jpuri**
Using new images for graphs

Screenshot 2022-01-24 at 6 11 32 AM

### CODE REVIEW

File: ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ const EditGasToolTip = ({
         editGasMode === EDIT_GAS_MODES.SWAPS
       ) &&
       !estimateGreaterThanGasUse ? (
-        <img alt="" src={`./images/curve-${priorityLevel}.svg`} />
+        <img alt="" src={`./images/curve-${priorityLevel}.png`} />
```
Comment (brad-decker) [inline ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js:91]: [REQ] Lets add an alt description using t('').

**Review by brad-decker — DISMISSED** [review]
Everything looks great and i love those graph images, just one accessibility nit that we should solve

**Review by mcmire — COMMENTED** [review]
Is it possible to keep these as SVGs? I think that would be more maintainable in the long run. If we use PNGs then we have to have Retina-compatible images so they appear crisp on Macs.

File: ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js (hunk 1 above)
Comment (jpuri) [reply ui/components/app/edit-gas-fee-popover/edit-gas-tooltip/edit-gas-tooltip.js]: Added the fix for this.

**Comment by jpuri** [comment]
@brad-decker , @mcmire : I did the fix to add alt text. For svg images I am checking with Jake.

**Review by mcmire — APPROVED** [review]
Looks better, thanks!

**Review by adonesky1 — APPROVED** [review]
LGTM!

**Pull request merged**